cmd/thanos/sidecar.go

```diff
@@ -506,7 +510,30 @@ type sidecarConfig struct {
 	storeRateLimits store.SeriesSelectLimits
 }
 
+type durationValue time.Duration
```

Comment:
We should be able to set duration via flag directly without implementing interface.

https://github.com/thanos-io/thanos/blob/main/cmd/thanos/receive.go#L968